### DISCUSSION TITLE
Downloaded books limit

### DISCUSSION THREAD

**Discussion by szy54** [tp86/legimi#13] [Q&A]
Hi, my legimi subscription is suspended at the moment, so I didn't check it myself.
Just out of curiosity, I was wondering if you checked whether the downloaded books limit (7 or 10) also applies when using the script?
It would be a silly mistake on legimi side, but nonetheless I was wondering whether the books limit information is kept on legimi servers or within the desktop app only.
Cheers,
Szymon

**Comment by tp86** [comment]
It's been a while since I checked how legimi works - I occassionally download a book or two within subscription period.

I haven't checked it explicitly, but I believe it's kept on servers. Why I think so:
1. There is an error "Unlimited download limit reached" which I think would be returned in such case.
2. There is already information if book was already downloaded by user available (and even displayed by script). While it seems that only this information is returned to client, it may be that full download information (download times, actually) is stored on server.

For sure, you can download the same book as many times as you want within single subscription period, I believe legimi checks how many **different** books were downloaded.

You may try (if your subscription is available again) and check and let know.
